tests/Ilios/MeSH/desc.xml

```diff
@@ -135,6 +135,8 @@ It contains all possible elements defined by the schema.
                 <CASN1Name>a casn1 name</CASN1Name>
                 <RegistryNumber>00000AAAAA</RegistryNumber>
                 <ScopeNote>a scope note</ScopeNote>
+                <TranslatorsEnglishScopeNote>something in English.</TranslatorsEnglishScopeNote>
+                <TranslatorsScopeNote>i got nothing</TranslatorsScopeNote>
```

Comment:
zero points for creativity!

Reply:
![zero point zero](https://www.mylespaul.com/data/xfmg/thumbnail/54/54566-a731eba88cebca8dd149fc68e01419ff.jpg?1552611134)